oms-active-directory-security-audit-solution/azuredeploy.json

```diff
@@ -0,0 +1,768 @@
+                                "Label": "Security Audit",
+                                
+                                
+                                "Link": {
+                                    "Label": "More info",
+                                    "Url": "https://asitha.info/"
+                                }
+                            },
+                            "List": [
+                                {
+                                    "Title": "Active Directory Security Audit Solution",
+                                    "Content": "## Users and Groups Monitoring View\n1. Created user accounts\n2. Deleted user accounts\n3. Group membership changes - Domain Admin and Local Admin\n4. Related quries\n \n\nPost your feedback and Let us know how to improve this solution - **asithaed@millenniumitesp.com**,\n\n#### Solution by [Asitha De Silva](https://www.linkedin.com/in/asitha-de-silva-429b0927/ \"Asitha De Silva\")"
+                                }
+                            ]
+                        }
+                    }
+                ],
+                "Filters": [],
+                "OverviewTile": {
+                    "Id": "DoubleNumberBuilderTile",
+                    "Type": "OverviewTile",
+                    "Version": 2,
+                    "Configuration": {
+                        "TileOne": {
+                            "Legend": "Account Created",
+                            "Query": "SecurityEvent | where (EventID == 624 or EventID == 4720) | summarize count() by SamAccountName | count"
+                        },
+                        "TileTwo": {
+                            "Legend": "Account Deleted",
+                            "Query": "SecurityEvent | where (EventID == 630 or EventID == 4726) | summarize count() by TargetUserName | count"
+                        },
+                        "Advanced": {
+                            "DataFlowVerification": {
+                                "Enabled": false,
+                                "Query": "search * | limit 1 | project TimeGenerated",
+                                "Message": ""
+                            }
+                        }
+                    }
+                }
+            }
+        },
+
+        {
+                
+                    "apiVersion": "[variables('LogAnalyticsApiVersion')]",

```

Comment:
use a literal apiVersion

Reply:
u mean to remove the variable and add the apiversion here ?

Reply:
done